CONTRIBUTING.md

```diff
@@ -1,29 +1,39 @@
 # Guidelines for contributing
 
-Thank you for your interest in contributing to AWS documentation! We greatly value feedback and contributions from our community.
+Welcome to AWS Docs! We thrive on your contribution. :heart:
 
-Please read through this document before you submit any pull requests or issues. It will help us work together more effectively.
+## Before you get started
+Please take 3 minutes to understand:
+1. [Submit an issue](#submit-an-issue)
+1. [What to expect from AWS](#what-to-expect-from-aws)
+1. [Write code and submit pull requests](#write-code-and-submit-pull-requests)
 
+---
 
+## Submit an issue
```

Comment:
I recommmend a single list, as in 

'We look forward to the following contributions:

* New examples for our documentation, such as examples for different platforms or environments, and examples in additional programming languages - for more information, see [Types of examples](https://github.com/awsdocs/aws-doc-sdk-examples/edit/main/CONTRIBUTING.md#types-of-examples) below.
* Corrections to inaccuracies in the content, including typos or grammatical errors
* Updates to information gaps in the content 
* Suggested rewrites that improve clarity and reduce confusion

**Note**: If you'd like to contribute, but don't have a project in mind, look at the [open issues](https://github.com/awsdocs/aws-doc-sdk-examples/issues) in this repository for some ideas. Any issues with the [help wanted](https://github.com/awsdocs/aws-doc-sdk-examples/labels/help%20wanted) or [enhancement](https://github.com/awsdocs/aws-doc-sdk-examples/labels/enhancement) labels are a great place to start.